### PR TITLE
ui: Tweak some code related meta information

### PR DESCRIPTION
**Pull request by johncowen**
This PR _mostly_ tweaks some code related meta information.

### CODE REVIEW

File: ui/packages/consul-ui/app/utils/get-environment.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default function(config = {}, win = window, doc = document) {
             .startsWith('CONSUL_')
         );
     };
-    win.Scenario = function(str = '') {
+    win['Scenario'] = function(str = '') {
```
Comment (randallmorey) [inline ui/packages/consul-ui/app/utils/get-environment.js:26]: Why this change?

**Review by randallmorey — APPROVED** [review]

File: ui/packages/consul-ui/app/utils/get-environment.js (hunk 1 above)
Comment (johncowen) [reply ui/packages/consul-ui/app/utils/get-environment.js:26]: `tsc` doesn't complain about it if I do this, and it's the best solution I could find given I want to write as little typescript as possible. It's not great I know but also importantly: it's wrapped in a runInDebug, so this is all compiled out of our production build, so I'm ok with it.

It's funny as I just realized that this is actually cheating a little as I wanted to see what was possible without changing any existing code, so I'm happy either way if this goes in or not.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/561712.
